_posts/0000-01-02-Billboard-4-0.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
so a rapper, a blind man, and a volcanologist go into the base of an active volcano...